frontend/components/FilterDisclosure.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+  <div class="w-full px-4 py-1">
+    <div class="w-full rounded-2xl">
+      <Disclosure v-slot="{ open }">
+        <DisclosureButton
+          class="flex w-full justify-center rounded-lg bg-slate-100 px-4 py-2 text-left text-sm font-medium text-slate-900 hover:bg-slate-200 focus:outline-none focus-visible:ring focus-visible:ring-slate-500 focus-visible:ring-opacity-75"
+        >
+          <span> {{ $t("CompanyBrowser.filter") }} </span>
+          <ChevronDownIcon
+            :class="open ? 'rotate-180 transform' : ''"
+            class="h-5 w-5 text-slate-500"
+          />
+        </DisclosureButton>
+        <DisclosurePanel class="px-4 pt-4 pb-2 text-sm text-slate-500">
+          <FilterCombobox
+            :placeholder="$t('Comboboxes.combobox-city')"
+            :items="cities"
+            :id="'cities-${index}'"
```

Comment:
Do we really need this `index`? IN this component there is no loop.

Reply:
It wasnt need it, already removed